ibm/resource_ibm_is_virtual_endpoint_gateway.go

```diff
@@ -118,13 +118,17 @@ func resourceIBMISEndpointGateway() *schema.Resource {
 						isVirtualEndpointGatewayIPsSubnet: {
 							Type:        schema.TypeString,
 							Optional:    true,
-							Computed:    true,
```

Comment:
why is this removed?

Reply:
Because.. there the get method is not having a parameter for `subnet` to set. 
https://github.com/VaishnaviGopal/terraform-provider-ibm/blob/14f456aceb64744c7e3be9fb0745704a87e08e0d/ibm/resource_ibm_is_virtual_endpoint_gateway.go#L401